CHANGELOG.md

```diff
@@ -11,6 +11,122 @@ git log --first-parent
 
 Please visit our wiki [Changelog](https://github.com/ginkgo-project/ginkgo/wiki/Changelog) for unreleased changes.
 
+## Version 1.10.0
+
+The Ginkgo team is proud to announce the new Ginkgo minor release 1.10.0.
+This release brings new features such as:
+- Support for bfloat16 precision. The type `gko::bloat16` can now be selected in most instances as the value type
+  of a matrix, solver, preconditioner, etc. If the selected backend supports bfloat16 as a native type, the native type
+  is used within the kernels, otherwise an overhead might occur. The new behavior is enabled by default, but it can be
```

Comment:
```suggestion
  is used within the kernels, otherwise they may incur a conversion overhead. The new behavior is enabled by default, but it can be
```